pkg/gen/input/workflows/internal/file/create_release.go

```diff
@@ -258,7 +258,7 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       GO_VERSION: 1.14.2
       ARTIFACT_DIR: bin-dist
-      PKG_VERSION: ${{ needs.gather_facts.outputs.version }}
+      PKG_VERSION: v${{ needs.gather_facts.outputs.version }}
```

Comment:
I like to think that the version is the thing w/o "v". And it's usually easier to add prefix than remove it in scripts so I think it would be good to keep the naming aligned.

```suggestion
      TAG: v${{ needs.gather_facts.outputs.version }}
```